wallet/src/de/schildbach/wallet/ui/WalletActivity.java

```diff
@@ -65,6 +65,8 @@
 
 import de.schildbach.wallet.livedata.Resource;
 import de.schildbach.wallet.livedata.Status;
+import de.schildbach.wallet.service.InactivityNotificationService;
```

Comment:
It seems that these imports are not being used.

Reply:
Removed: https://github.com/dashevo/dash-wallet/pull/393/commits/a97121de3c2191f9550a247870638dcadf76b503